cli/command/formatter/container_test.go

```diff
@@ -390,7 +390,7 @@ func TestContainerBackCompat(t *testing.T) {
 	for _, c := range cases {
 		ctx := Context{Format: Format(fmt.Sprintf("{{ .%s }}", c)), Output: buf}
 		if err := ContainerWrite(ctx, containers); err != nil {
-			t.Log("could not render template for field '%s': %v", c, err)
+			t.Fatalf("could not render template for field '%s': %v", c, err)
 			t.Fail()
```

Comment:
This way we can see all failures in a single test run.

Realisticly, changing this to the new go1.7 sub-test would be better, but at the moment we still have people compiling with go1.6.